charts/prometheus-elasticsearch-exporter/README.md

```diff
@@ -0,0 +1,86 @@
+# Elasticsearch Exporter
+
+Prometheus exporter for various metrics about ElasticSearch, written in Go.
+
+Learn more: <https://github.com/justwatchcom/elasticsearch_exporter>
+
+This chart creates an Elasticsearch-Exporter deployment on a [Kubernetes](http://kubernetes.io)
+cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+
+## Get Repo Info
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm 3
+$ helm install [RELEASE_NAME] prometheus-community/prometheus-stackdriver-exporter --set stackdriver.projectId=google-project-name
```

Comment:
```suggestion
$ helm install [RELEASE_NAME] prometheus-community/prometheus-elasticsearch-exporter [flags]
```

Reply:
I think this was there by intention as deploying the chart without it does not make sense.

Reply:
Was to late yesterday ;-) It's fixed now.